sw/host/opentitanlib/src/spiflash/sfdp.rs

```diff
@@ -502,4 +510,18 @@ mod test {
         assert_eq!(sfdp.jedec.density, 128 * 1024 * 1024);
         Ok(())
     }
+
+    // Regression test for https://github.com/lowRISC/opentitan/issues/13477
```

Comment:
nit: If you want you can  use one of the following shorthand(s) in the future: `#13477` or `lowRISC/opentitan#13477`

Reply:
Noted! The downside of that shorter notation is it's no longer a URL. Chrome has a custom link shortener for bugs, so you can link to `crbug.com/1234`. Maybe we should think about setting up something like that.

Reply:
If you don't mind, I'll leave this as URL for ease of clicking. Especially since it fits on one line.